test/test_autograd.py

```diff
@@ -2285,6 +2285,15 @@ def rfft_irfft(x):
         _test_complex((1, 2, 3, 4, 2), 2)
         _test_complex((2, 1, 3, 4, 3, 2), 3)
 
+    def test_gradcheck(self):
```

Comment:
please name this better -- there are ton of things that we could check about gradcheck and this is one single example.

Reply:
Fixed